io.gitlab.Turtlico.json

```diff
@@ -10,14 +10,19 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
         "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
         "--device=all",
-        "--env=PYTHONPATH=/usr/lib/python3.8/site-packages",
-        "--env=GPIOZERO_PIN_FACTORY=native"
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=GPIOZERO_PIN_FACTORY=native",
+        "--env=PYTHONUNBUFFERED=1"
     ],
     "cleanup": [
         "/include",
         "/lib/pkgconfig",
+        "/lib/debug",
```

Comment:
no. don't delete this, it is put into a separate `.Debug` package.

Reply:
Ok. This is probably from a manifest template file that I used.